js/addnew.js

```diff
@@ -0,0 +1,24 @@
+class Book {
+  constructor(name, author) {
+    this.name = name;
+    this.author = author;
+  }
+}
+
+let bookList = JSON.parse(localStorage.getItem('books'));
+if (bookList == null) {
+  bookList = [];
+}
+
+function addNewBook(book) {
+  bookList.push(book);
```

Comment:
[ ] kindly specify a primary key for your book, it could be an id, title, or the author but it should be unique. (i created two books with the same name and when I deleted one of them, both are deleted. kindly fix that  🤓). 
`hint: you can generate an id when you create a new book, and in the delete method you can filter the array using that id to remove the book.`

![unique](https://user-images.githubusercontent.com/62156607/197637806-c368f7a4-c7f1-4de5-ae7b-cb910ba3d993.PNG)